src/main/java/com/parasoft/findings/sonar/MapperUtil.java

```diff
@@ -57,9 +59,13 @@ public static SoftwareQuality mapToSonarImpactSoftwareQuality(String category, b
 
     public static CleanCodeAttribute mapToSonarCleanCodeAttribute(String category) {
         String tag = categoryToTag(category);
-        // TODO will be implemented in CICD-666
-        // The following return value is just for testing, because the default value is CleanCodeAttribute.CONVENTIONAL if not set.
-        return CleanCodeAttribute.CLEAR;
+        String[] splitTag = tag.split("\\.");
+        if (TagPrefixMap.TAG_PREFIX_MAP.containsKey(splitTag[0])) {
+            return TagPrefixMap.TAG_PREFIX_MAP.get(splitTag[0]);
+        } else {
+            Logger.getLogger().warn("The category " + category + " can not found in Parasoft rule categories, map it to CONVENTIONAL."); //$NON-NLS-1$
```

Comment:
Let's modify this to be a debugging message, as it's intended for us, not the users.
Please change it to:
`The category 'xxx' has been mapped to 'CONVENTIONAL' because it does not match any existing categories in the plugin.`